src/components/Common.js

```diff
@@ -37,6 +38,14 @@ export const Button = styled.a`
   padding: 0.75em 2.5em;
   border-radius: 3px;
   margin: 1em;
+
+  ${p =>
+    p.theme.custom_imgs === 'hc' &&
+    css`
```

Comment:
same thing here?

Reply:
this file already imports css so i think it should be fine lol